core/src/main/java/com/bankbazaar/tripmanager/model/Trip.java

```diff
@@ -44,4 +44,28 @@ public class Trip implements Serializable {
     @LastModifiedDate
     @Column(name = "last_modified_date")
     private Date lastModifiedTime;
+
+    public void updateData(Trip trip)
```

Comment:
Model classes should not have this. This should be in manager

Reply:
this method has been moved to manager class